docs/attach-setup.asciidoc

```diff
@@ -0,0 +1,193 @@
+[[attach-setup]]
+== Setup the agent with the attacher
+
+You can use the attacher as a standalone application on the command line or include it in your application to programmatically attach to the current JVM.
+
+This installation method does not require you to alter the configuration of your application server and can be used to conveniently instrument all JVMs on a particular host.
+
+NOTE: This installation method is experimental.
+
+[float]
+[[attach-setup-cli]]
+=== Use the attacher as a command line application
+
+The `apm-agent-attach.jar` is a small Java program which attaches the Elastic APM Java agent to a specific JVM or to all JVMs of the same host it runs on.
+This requires that the OS user of the attach is the same as the OS user of the applications to be monitored (the JVMs to attach to).
+
+
+[float]
+[[attach-setup-cli-download]]
+=== Download
+
+You can download the attach program from maven central:
+link:https://search.maven.org/search?q=g:co.elastic.apm%20AND%20a:apm-agent-attach[maven central]
```

Comment:
Should this link work yet? It just brings me to a blank search page

Reply:
That link will only work once this change gets released.